Source: Upstream
Upstream: yes
Reason: Fix compile issue when inotify is disabled
diff --git a/src/Thumbview.cc b/src/Thumbview.cc
index 0b85974..a01d893 100644
--- a/src/Thumbview.cc
+++ b/src/Thumbview.cc
@@ -745,6 +745,8 @@ void Thumbview::file_created_callback(std::string filename) {
 	}
 }
 
+#endif
+
 void Thumbview::set_current_display_mode(DisplayMode newmode)
 {
     remove();
@@ -776,4 +778,4 @@ void Thumbview::sighandle_view_activated(const Gtk::TreePath& path, Gtk::TreeVie
     signal_selected(path);
 }
 
-#endif
+
